packages/babel-template/src/index.js

```diff
@@ -99,6 +99,7 @@ let templateVisitor = {
   },
 
   exit({ node }) {
-    traverse.clearNode(node);
+    if (!node.file)
```

Comment:
Well it fixed the errors at https://travis-ci.org/babel/babel/jobs/106854449#L449!